BuildDetails.md

```diff
@@ -0,0 +1,55 @@
+This is the build description guide that explains the build process for all FullScreenShenanigans projects. The guide will explain to you how projects are built and why they are built that way. Please note that thes guide only fully applies to the GameStartr games, although some of it can be applied to the modules as well.
```

Comment:
It would be better to say that most tasks are used in the modules as well. 

It's a little more precise / less ambiguous, and indicates that knowledge here fully applies there.